documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -94,10 +105,52 @@ m¦excludedBrokersForReplicaMove
 
 |===
 
+
+[discrete]
+=== Broker load
+
+The broker load is stored in a ConfigMap (with the same name as the KafakRebalance custom resource) as a JSON formatted string. This JSON string consists of a JSON object with keys for each broker IDs linking to a number of metrics for each broker.
+Each metric consist of three values.
+The first is the metric value before the optimization proposal is applied, the second is the expected value of the metric after the proposal is applied, and the third is the difference between the first two values (after minus before).
+
+To extract the JSON string from the ConfigMap, use the following command:
```

Comment:
`jq` is just one option for doing this and we don't list `jq` as something you *need* in order to use Strimzi. So I would change this to something like: 

> To extract the JSON string from the ConfigMap you could use the following command, which uses the `jq` command line JSON parser tool:

Reply:
Sounds good.